web/src/elements/table/Table.ts

```diff
@@ -74,18 +74,11 @@ export abstract class Table<T extends object>
                 .presentational {
                     --pf-c-table--cell--MinWidth: 0;
                 }
-                @container (width > 600px) {
+                @container (width > 1200px) {
```

Comment:
The previous number was just as arbitrary but lacked consideration for some of our busier pages like the user's list.